src/routes/store/gift-cards/[slug]/+page.svelte

```diff
@@ -120,11 +135,23 @@
 			</div>
 
 			<div class="mt-5 rounded-t-lg bg-brand-100 p-4">
-        {#if prod_desc.price_tags?.denominations?.length}
+        {#if prod_desc.price_tags?.denominations?.length || prod_desc.price_tags.flexible}
```

Comment:
To prevent accidental crashes in case of malformed data from your api, you should always use optional chaining `?.` when nesting objects greater than one depth.

This will prevent ```trying to access property x of undefined``` errors when the data is incomplete